src/components/Form.js

```diff
@@ -0,0 +1,13 @@
+export default function Form({ handleSubmitForm, handleOnChangeName }) {
```

Comment:
When we make a component receive props to handle "events", we call them `onSubmit` or `onChangeName`

I'm not sure you need both here though. Think that the only thing you want is that when the form is submitted, the parent component can handle that and receive the name that was written in the form.

You can take a look to debs implementation